external/kafka-0-10-token-provider/src/test/scala/org/apache/spark/kafka010/KafkaTokenUtilSuite.scala

```diff
@@ -233,8 +233,27 @@ class KafkaTokenUtilSuite extends SparkFunSuite with KafkaDelegationTokenTest {
     assert(jaasParams.contains(tokenPassword1))
   }
 
-  test("isConnectorUsingCurrentToken without security should return true") {
-    val kafkaParams = Map[String, Object]().asJava
+  test("isConnectorUsingCurrentToken without security credentials enabled should return true") {
+    setSparkEnv(
+      Map(
+        s"spark.security.credentials.kafka.enabled" -> "false"
+      )
+    )
+    val kafkaParams = getKafkaParams(true, Some("custom_jaas_config"))
```

Comment:
I'd add boolean param name explicitly if it doesn't exceed line length limit, as it's not easy to get the meaning. Same applies under all calling getKafkaParams.

Reply:
Added.